app/controllers/idv/in_person/address_search_controller.rb

```diff
@@ -0,0 +1,39 @@
+require 'json'
+
+module Idv
+  module InPerson
+    class AddressSearchController < ApplicationController
+      include ArcgisApi
+
+      def geocoder
+        @geocoder ||= Geocoder.new
+      end
+
+      def suggest(address)
+        geocoder.suggest(address)
+      end
+
+      def find_address_candidates(magic_key)
+        geocoder.find_address_candidates(magic_key)
+      end
+
+      def index
+        arcgis_api_response = []
+        begin
+          suggestion = suggest(permitted_params[:address]).first
+          arcgis_api_response = [find_address_candidates(suggestion.magic_key).first]
```

Comment:
We probably want to rate limit these external calls in some form, but I'm not sure on the best strategy there

Reply:
Yep, I think that would be the path to take. It should probably be implemented within the service rather than the controller though.